backend/src/main/resources/db/migration/V6_fcms_v2_changes.sql

```diff
@@ -0,0 +1,21 @@
+alter table nrfc.submission_detail
+add column if not exists work_safe_bc_number varchar(6) null,
+add column if not exists doing_business_as varchar(120) null,
+add column if not exists client_acronym varchar(8) null,
+add column if not exists TBD_ID_TYPE varchar(8) null,
+;
+
+comment on column nrfc.submission_detail.work_safe_bc_number is 'TBD';
```

Comment:
Shouldn't we add a proper comment here? If there's no info to be added, is better to remove it at all

Reply:
Yes, this is a draft. I am working on it. I will request your review once I am done.